eng/Versions.props

```diff
@@ -129,7 +129,7 @@
     <MicrosoftVisualStudioCacheVersion>17.2.41-alpha</MicrosoftVisualStudioCacheVersion>
     <MicrosoftVisualStudioCallHierarchyPackageDefinitionsVersion>15.8.27812-alpha</MicrosoftVisualStudioCallHierarchyPackageDefinitionsVersion>
     <MicrosoftVisualStudioCodeAnalysisSdkUIVersion>15.8.27812-alpha</MicrosoftVisualStudioCodeAnalysisSdkUIVersion>
-    <MicrosoftVisualStudioComponentModelHostVersion>17.2.3194</MicrosoftVisualStudioComponentModelHostVersion>
+    <MicrosoftVisualStudioComponentModelHostVersion>17.3.37-preview</MicrosoftVisualStudioComponentModelHostVersion>
```

Comment:
@jmarolf can this go into main?  not sure our policy on preview packages.

Reply:
also, what was thsi change needed for @emilyanas2323 ?

Reply:
> @jmarolf can this go into main? not sure our policy on preview packages.

Integration tests are passing so there is no breaking changes for contributors.

Reply:
As long as you can F5 from the last public preview it should be fine to go into main.

Reply:
@jmarolf do you know what this change was needed for? I'm not sure...

Reply:
Yep, confirmed this is good to go in. Public preview has a newer version of this assembly

Reply:
@CyrusNajmabadi I am going to close this as "resolved" since this is safe to take into main.